### PR TITLE
add stripe

### PR DESCRIPTION
**Pull request by jolly-sy**
Signed-off-by: jolly-sy <757050468@qq.com>

### What problem does this PR solve?

Issue Number: #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by jolly-sy** [comment]
recheck

File: src/tools/createtool.cpp [hunk 1]
```diff
@@ -33,6 +33,8 @@
 DEFINE_string(mds_addr, "127.0.0.1:9000", "mds addr");
 DEFINE_string(file_name, "vdisk_001", "file name");
 DEFINE_uint64(file_size, 1 * 1024 * 1024 * 1024, "file size");
+DEFINE_uint64(stripe_Unit, 32 * 1024, "stripe unit size");
```
Comment (wu-hanqing) [inline src/tools/createtool.cpp:36]: it seems this file is no longer in use, you can delete it.

File: src/tools/mds_client.h [hunk 2]
```diff
@@ -172,7 +172,8 @@ class MDSClient {
      *  @param length 文件长度
      *  @return 成功返回0，失败返回-1
      */
-    virtual int CreateFile(const std::string& fileName, uint64_t length);
+    virtual int CreateFile(const std::string& fileName, uint64_t length,
```
Comment (wu-hanqing) [inline src/tools/mds_client.h:175]: add comments for new arguments

File: src/tools/namespace_tool_core.h [hunk 3]
```diff
@@ -107,7 +107,8 @@ class NameSpaceToolCore {
      *  @param length 文件长度
      *  @return 成功返回0，失败返回-1
      */
-    virtual int CreateFile(const std::string& fileName, uint64_t length);
+    virtual int CreateFile(const std::string& fileName, uint64_t length,
```
Comment (wu-hanqing) [inline src/tools/namespace_tool_core.h:110]: ditto, add comments

File: src/tools/namespace_tool.cpp [hunk 4]
```diff
@@ -40,6 +40,8 @@ DEFINE_string(throttleType, "", "throttle type");
 DEFINE_uint64(limit, 0, "throttle limit");
 DEFINE_int64(burst, -1, "throttle burst");
 DEFINE_int64(burstLength, -1, "throttle burst length");
+DEFINE_uint64(stripeUnit, 32 * 1024, "stripe unit size");
+DEFINE_uint64(stripeCount, 32, "strip count");
```
Comment (wu-hanqing) [inline src/tools/namespace_tool.cpp:44]: should we take these default values? @ilixiaocui

**Pull request closed without merging**